Yoast/YoastSEO.js
# Improve the keyword in subheadings assessment

## Summary

This PR can be summarized in the following changelog entry:
* Improves the keyword in subheadings assessment so that it checks whether the keyword is included in too few or too many subheadings with regards to the total number of subheadings in the text.

## Relevant technical choices
* Adds missing translator comments to `metaDescriptionKeywordAssessment`

## Test instructions

This PR can be tested by following these steps:

* Check out the branch.
* Use the browserified example. Don't forget to run a `grunt build:js`.
* Add a text and a keyword.
* Include some subheadings in your text. Try various combinations with subheadings containing the keyword and subheadings not containing the keyword. Make sure the plugin provides meaningful feedback strings.

Note: This assessment is currently still triggered when using multiple keywords with Premium. However, this won't be the case anymore once a secondary keyword assessor has been implemented (https://github.com/Yoast/YoastSEO.js/issues/1479) in order to prevent keyword stuffing.

Fixes #1311
